coalib/bears/Bear.py

```diff
@@ -461,13 +461,18 @@ def execute(self, *args, debug=False, **kwargs):
                          .format(name), str(exc))
 
             if self.kind() == BEAR_KIND.LOCAL:
-                self.err('Bear {} failed to run on file {}. Take a look '
-                         'at debug messages (`-V`) for further '
-                         'information.'.format(name, args[0]))
+                if self.section['log_level'].value == 'DEBUG':
+                    pass
+                else:
+                    self.err(
+                        'Bear {} failed to run on file {}. Take a look '
+                        'at debug messages (`-V`) for further '
+                        'information.'.format(name, args[0]))
             else:
-                self.err('Bear {} failed to run. Take a look '
```

Comment:
Unrelated change, revert it back.

Reply:
The logic is not correct here. Your code is saything that If `self.kind() == BEAR_KIND.LOCAL` and `self.section['log_level'].value == 'DEBUG'`, display error message for local bear. Otherwise, display error message for global bear.

Reply:
@li-boxuan @RaiVaibhav  i had already updated my pr but you haven't reviewed please review